### PR TITLE
Remove manual nat manager

### PR DESCRIPTION
**Pull request by matkt**
Signed-off-by: Karim TAAM <karim.t2am@gmail.com>

## PR description

We decided to add a manual mode during the implementation of the NatManager.  This mode does not seem really useful because it does the same thing as a NONE mode (it takes the parameters p2p-host, p2p-port, etc). At the moment this mode does not bring much and seems to be confusing. To avoid confusion the best is to delete ManualNatManager

This requires an update of the documentation (https://besu.hyperledger.org/en/stable/HowTo/Find-and-Connect/Specifying-NAT/#manual)
- Remove the MANUAL part
- Change NONE part
````
By specifying NONE you can set the following parameters if needed:
--p2p-host and --p2p-port define the advertised host and port for the P2P service.
--rpc-http-host and rpc-http-port define the advertised host and port for the JSON-RPC service.
````

### CODE REVIEW

**Comment by matkt** [comment]
> I forgot to do this in my PR so just reminding you here to write an entry in the changelog in this PR :) It will help out the docs team so they don't have to do this last minute when the release is happening.


I added it in the `Upcoming 1.5 release` section of the CHANGELOG. feel free to tell me if it's not ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ is owned by that user.
 
 The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July. 
 
+### Additions and Improvements
+
+- Remove MANUAL nat manager method. It is now necessary to use the NONE method which has the same behavior. [\#1041](https://github.com/hyperledger/besu/pull/1041)
```
Comment (RatanRSur) [inline CHANGELOG.md:27]: Let's put this in the "Breaking Changes" section above instead.

Comment (matkt) [reply CHANGELOG.md]: Done

**Comment by MadelineMurray** [comment]
Doc'd https://besu.hyperledger.org/en/latest/HowTo/Find-and-Connect/Specifying-NAT/

**Pull request merged**